### DISCUSSION TITLE
Manually update version of installed application

### DISCUSSION THREAD

**Discussion by mrbusche** [ScoopInstaller/Main#4079] [Q&A] [answered]
I'm trying to update cmder but my Corporate firewall is flagging it as a virus. I was able to obtain the zip file, unzip, and place in the cmder folder as `1.3.20` and replaced the contents in `current` but it appears there's another place in scoop I need to update, so it knows I'm on the latest version already. Anyone know where that is?

**Comment by niheaven** [comment, marked as answer]
Rename downloaded file to `cmder#1.3.20#https_github.com_cmderdev_cmder_releases_download_v1.3.20_cmder_mini.zip` and put it under `cache` folder of scoop, then you could `scoop update cmder`.

**Reply by mrbusche** [reply]
That worked, thanks a lot!
